srcpkgs/python3-rdflib/patches/setup.patch

```diff
@@ -0,0 +1,9 @@
+--- a/setup.py
++++ b/setup.py
+@@ -88,5 +88,5 @@
+             "rdfgraphisomorphism = rdflib.tools.graphisomorphism:main",
+         ],
+     },
+-    **kwargs,
++    **kwargs
```

Comment:
Why is this change necessary? The interpreter should have no problem with `**kwargs,` here.

Reply:
not sure, it `SyntaxError`s

Reply:
Not for me...

Reply:
weird, it was happening for me on multiple versions when I made that.